tasks/install_modjk.yml

```diff
@@ -29,15 +29,15 @@
     chdir: "{{ apache_modjk_build_dir }}"
 
 - name: Apache_httpd | Copy mod_jk to Apache modules folder
-  copy:
+  template:
```

Comment:
is the so (shared object) file a template?

Reply:
Nope, it's a mistake, I wanted to change to template L40 and L49 and, for some reason, I changed L32 instead of L49.